DESCRIPTION

```diff
@@ -31,13 +31,15 @@ VignetteBuilder:
     knitr
 Encoding: UTF-8
 Language: en-US
+Depends: 
+    R (>= 3.6.0)
```

Comment:
@const-ae Since ggsignif can't work without ggplot2, and since ggplot2 theoretically now supports only R > 4.0, I'd actually prefer to set this to `R 4.0`. WDYT?

https://www.tidyverse.org/blog/2019/04/r-version-support/

<img width="807" alt="Screenshot 2024-05-05 at 17 54 31" src="https://github.com/const-ae/ggsignif/assets/11330453/b0a0c744-2b6d-4cce-a612-4e70ab6f3be2">

Reply:
Hmm, I don't like forcing a minimum version without good reason. People who are stuck on an old R version and refrain from upgrading ggplot2 should still be able install `ggsignif` from CRAN. 
If, at some point, we need to adapt `ggsignif` because `ggplot`'s internals change, we should reflect that with a minimum version for `ggplot`. I would only set the minimum version if we use features only present in recent versions of R (e.g., short-hand notation for functions or the native pipe).

Reply:
But, whether we specify a minimum version in `DESCRIPTION` or not, it is there. Nobody can install `{ggsignif}` if they can't even install `{ggplot2}`. So, for all intents and purposes, we already do have a minimum version; it's just invisible.

The good reason is that we [can't guarantee](https://indrajeetpatil.github.io/preventive-r-package-care/#/the-unbearable-diversity-of-contexts) that the package is working as expected on these older versions because neither CRAN nor our CI checks the package on these older versions. So any changes we have made recently might already broken the package for (say) R 3.5 and we wouldn't even know about it, unless, of course, you check the package on these older R versions before CRAN submission.